post-install.sh

```diff
@@ -17,7 +17,7 @@ setup_command=${cfn_postinstall_args[2]}
 monitoring_home="/home/${cfn_cluster_user}/${monitoring_dir_name}"
 
 case ${cfn_node_type} in
-    HeadNode)
+    MasterServer)
```

Comment:
Let's make this apply to all versions i.e.

```bash
case ${cfn_node_type} in
     HeadNode | MasterServer)
```

Reply:
Nice!  I was wondering maybe this was multi-modal.